src/ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -506,10 +506,15 @@ def _connect(self, timeout=5, set_no_abort=True, enable_health_check=False):
             self._timer.start()
 
         # initialize mesh, post processing, and file explorer interfaces
-        from ansys.mapdl.core.mesh_grpc import MeshGrpc
+        try:
+            from ansys.mapdl.core.mesh_grpc import MeshGrpc
+
+            self._mesh_rep = MeshGrpc(self)
+        except:
```

Comment:
Should not have bare excepts.

If it can't be tested in our framework, ignore coverage.

```suggestion
        except ImportError:  # pragma: no cover
```